sprocs/access_tokens_delete.sql

```diff
@@ -15,7 +15,7 @@ BEGIN
         a.* INTO old_row;
 
     INSERT INTO audit_logs
-        (user_id, table_name,
+        (authn_user_id, table_name,
```

Comment:
I think we should actually store both `user_id` and `auth_user_id` into the two columns with those names in `audit_logs`.

Reply:
Are they ever going to be different on this page? IIRC you can't have an effective user outside the context of course instances.

Reply:
Say an instructor changes their effective user id (`user_id`) to that of a TA in the class, and then creates/deletes access tokens for the TA. Then `authn_user_id` would be the instructor and `user_id` would be the TA.

Reply:
Right, what I'm saying is that I can't find a way to make the effective user that I set on a course instance take effect on the settings page. I can become person X in a course instance, but once I go back to the homepage or visit my settings page, my identity reverts to its actual one.

Reply:
Oh right, good point, I think the settings page is special in this respect. In this case I agree it’s not necessary to distinguish the two types of user ids, although we could always store both for completeness :-)

Reply:
Good idea, that'll potentially make querying easier in the future. Done 🙂